apiv2/schema/schema.yaml

```diff
@@ -2148,6 +2148,42 @@ classes:
           - tiltseries_aligned_tiltseries_binning
         range: integer
         minimum_value: 0
+      size_x:
+        name: size_x
+        description: Number of pixels in the 3D data fast axis
+        from_schema: cdp-dataset-config
+        exact_mappings:
+          - api_tiltseries_size_x
+        range: integer
+        required: false
+        minimum_value: 0
+        unit:
+          symbol: voxel
```

Comment:
I think the x and y axis units for tiltseries is `pixel` (symbol: 'px'rather than `voxel` (since tiltseries aren't volumes yet) --

```
        unit:
          symbol: px
          descriptive_name: pixels
```
I don't think the z-axis has units, we probably want to omit units there (cc @uermel)